### ISSUE TITLE
UpgradeDependencyVersion inserting properties aren't formatted correctly

### ISSUE THREAD

**Issue by jkschneider** [openrewrite/rewrite#1473]
### Problem

`<jackson.version>` isn't indented correctly. Also this shouldn't resort all tags. Note that the source encoding tag got bumped to the end of the set of properties here.

cc / @tkvangorder 

### Example diff

```diff
     <url>https://thingsboard.io</url>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <jackson.version>2.12.6</jackson.version>
         <main.dir>${basedir}/../..</main.dir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
     <dependencies>
```

#### Recipes in example diff:
- org.openrewrite.maven.UpgradeDependencyVersion

### References:
- Recipe ID: `4aNFQ`
- Recipe Name: "Upgrade Maven dependency version"
- Repository: thingsboard/thingsboard/master
- Created at Wed Mar 16 2022 17:49:22 GMT-0400 (Eastern Daylight Time)

**Comment by tkvangorder** [comment]
The indention of a property in the unit tests is working as expected, I am going to deploy the new snapshot to the SaaS and see if the indention issue is still present on this specific repo.

**Comment by tkvangorder** [comment]
XML Formatting has been re-implemented (See #1477). Closing this issue.
